src/appleseed.python/bindblendermeshconverter.cpp

```diff
@@ -35,81 +35,76 @@
 namespace bpy = boost::python;
 using namespace renderer;
 
-// Blender data structures
-struct MeshFace
+namespace
 {
-    unsigned int v[4];
-    short mat_nr;
-    char edcode, flag;
-};
+    // Blender data structures
+    // https://developer.blender.org/diffusion/B/browse/master/source/blender/makesdna/DNA_meshdata_types.h
+    struct MeshFace
+    {
+        unsigned int v[4];
+        short mat_nr;
+        char edcode, flag;
+    };
 
-struct MeshVert
-{
-    float co[3];
-    short no[3];
-    char flag, bweight;
-};
+    struct MeshVert
+    {
+        float co[3];
+        short no[3];
+        char flag, bweight;
+    };
 
-struct MeshTexFace
-{
-    float uv[4][2];
-    void *tpage;
-    char flag, transp;
-    short mode, tile, unwrap;
-};
+    struct MeshTexFace
+    {
+        float uv[4][2];
+        void *tpage;
+        char flag, transp;
+        short mode, tile, unwrap;
+    };
+}  // namespace
 
 
 // The following function takes a series of pointers to Blender mesh data and modifies the appleseed MeshObject entity
 //      blender_object = the appleseed MeshObject created earlier in the export process
-//      vertices_length = the number of vertices in the mesh
+//      vertices_size = the number of vertices in the mesh
 //      vertex_pointer = a string pointer to the first element of the vertex array
 //      Same template applies to mesh faces
 
 void convert_bl_mesh(
-    MeshObject*                 blender_mesh,
-    const size_t                vertices_length,
-    const size_t                vertex_pointer,
-    const size_t                triangle_length,
-    const size_t                triangle_pointer,
-    const size_t                uv_layer_pointer,
-    const bool                  export_normals,
-    const bool                  export_uvs)
+    MeshObject*                     blender_mesh,
+    const size_t                    bl_vert_size,
+    const uintptr_t                 bl_vert_ptr,
```

Comment:
Make sure to include `<cstdint>` for `uintptr_t`.

Reply:
Done